docs/testrunner-toolkit.md

```diff
@@ -136,6 +136,7 @@ Browser support for each framework is based on the Sauce Labs docker images prov
 
 |Cypress Version|Supported Browsers|
 |----|----|
+|7.3.0|Please see [release notes](https://github.com/saucelabs/sauce-cypress-runner/releases/tag/v7.1.0)|
```

Comment:
Is the link correct? It says v7.1.0

Reply:
It is, version number of runner is not in sync with version of the framework (but it can be misleading, you're right :))

Reply:
Cool. Just wanted to check!